docs/book/src/user/quick-start.md

```diff
@@ -93,6 +93,45 @@ a target [management cluster] on the selected [infrastructure provider].
    Then follow the instruction for your kind version using  `kind create cluster --config kind-cluster-with-extramounts.yaml`
    to create the management cluster using the above file.
 
+   {{#/tab }}
+   {{#tab KubeVirt}}
+
+   #### Create the Kind Cluster
+   To allow using LoadBalancer, we can't use the kind's default CNI (kindnet), but we'll need to install another CNI, like Calico. In order to do that, we'll need first to initiate the kind cluster with two modifications:
```

Comment:
I think it could be useful to add a note explaining why it is required to have a load balancer in the management cluster (which is something we do not have for other providers)?